pkg/agent/cniserver/secondary.go

```diff
@@ -56,6 +62,15 @@ func (pc *podConfigurator) ConfigureSriovSecondaryInterface(
 	containerConfig := buildContainerConfig(hostInterfaceName, containerID, podName, podNamespace,
 		containerNetNS, containerIface, result.IPs, 0)
 	pc.ifaceStore.AddInterface(containerConfig)
+	sriovInterface := &interfacestore.SRIOVInterfaceConfig{
+		InterfaceConf: containerConfig,
+		VFDeviceID:    podSriovVFDeviceID,
+		ResourceName:  resourceName,
+	}
+	pc.sriovIfaceStore.Save(filestore.AnyObjectWithUID{
```

Comment:
do you think usage of the `sriovIfaceStore` could be moved entirely to `secondarynetwork/podwatch` ? In general, I think it's better if we keep most of the logic there, and we rely on the event channel for communications between the CNI server and the SecondaryNetwork Pod controller.